src/components/search.tsx

```diff
@@ -11,13 +13,60 @@ export enum SearchType {
 }
 
 interface Props {
-  setSearch: (search: string) => void;
-  setSearchType: (searchType: SearchType) => void;
+  getSearch: (search: string) => void;
+  getSearchType: (searchType: SearchType) => void;  
+  getResultsImgs: (resultsImgs: Image[]) => void;
 }
 
-export const Search: React.FC<Props> = ({ setSearch, setSearchType }) => {
-  const [val, setVal] = useState("");
```

Comment:
There's no need to delete this, this is used to prevent a search on each keystroke, which results in way too many requests to the backend.